lib/resources/r4/observation.yaml

```diff
@@ -117,6 +178,63 @@ fields:
     }
   url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.code
 
+- name: code
+  required: 'Yes'
+  type: CodeableConcept
+  description: Describes what was observed. Sometimes this is called the observation "name".
+  action:
+    - terminology
+    - update
+  binding:
+    description: Codes identifying names of simple observations.
+    terminology:
+    - display: LOINC
+      system: http://loinc.org
+      info_link: https://hl7.org/fhir/R4/loinc.html
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+      info_link: https://hl7.org/fhir/R4/snomedct.html
+    - display: Millennium Clinical Event Code
+      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/72
+      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-72-clinical-event-codes
+  example: |
+    {
+      "code": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/72",
+            "code": "21704910",
+            "display": "Potassium Level",
+            "userSelected": true
+          }
+        ],
+        "text": 'Potassium Level'
+      }
+    }
+  example2: |
+    {
+      "code": {
+        "coding": [
```

Comment:
we don't support multiple codings, right?

Reply:
we don't. May be example 2 can have just one coding with 'loinc' system.

Reply:
+1

Reply:
updated the example: https://github.com/cerner/fhir.cerner.com/commit/be02d7b53316e773b9fbd31f46412de7954ed7a3#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcL218